about.html

```diff
@@ -0,0 +1,102 @@
+            <img src="images/twitter.svg" alt="" class="twitter-image">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </div>
+    </header>
+
+    <header class="hamburger-logo">
+
+        <div class="header-container">
+            <div><a href="index.html"><img src="images/mobile_menu1.svg" class="ham1"></a></div>
+            <nav>
+                <img src="images/F1_logo.png" class="logo">
+            </nav>
+        </div>
+
+        <nav class="menu" id="menu-pop">
+            <img src="images/Icon-close.svg" alt="" class="close-button">
+            <ul class="nav-content">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#program" id="nav-grey">Program</a></li>
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+            </ul>
+        </nav>
+
+    </header>
+
+    <section>
+        <div class="about-front">
+            <div class="about-intro">
+                <p id="about-hello">"Hello!,Sharing World"</p>
+                <div>
+                    <h1 class="about-title-intro">FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+                </div>
+            </div>
+
+            <div class="global-intro">
+                <p>The Global Summit brings together the community of  
+                experts,academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals.</p>
+            </div>
+            <div class="about-contact">
+                <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+                <p id="about-email">ccsummit2015@cckorea.org</p>
+            </div>
+
+            <div class="about-crts">
+                <p>Creative Commons Global Summit 2015 Logo</p>
+                <hr class="about-hr">
+                <p class="att-dates">The logo of CC Global Summit 2015 was decided through the 
+                    logo competition from 8, June to 7, July.</p>
+
+                <hr class="about-sm-line">
+                <img src="images/about_logo 1.svg" class="about-logo">
+            </div>
+        </div>
+    </section>
+
+    <section>
+        <div class="about-last">
+            <p class="about-cc-summit">See the past CC Global Summits</p>
+            <hr class="about-hr2">
+            <p class="about-look">Take a look at the past two CC Global Summits which took place
+                 in Buenos Aires and in Warsaw</p>
+        </div>
+
+        <div class="about-images">
+            <img src="images/about_2011 1.svg" alt="" class="fx-about-2011">
+            <img src="images/about_2013 1.svg" alt="" class="fx-about-2011">
+        </div>
```

Comment:
- [ ] For semantic reason ,kIndly make sure all the `alt` attributes are not empty 👍

Reply:
Added alt attributes